com.discordapp.DiscordCanary.json

```diff
@@ -49,14 +55,15 @@
             "name": "discord",
             "buildsystem": "simple",
             "build-commands": [
-                "install -Dm755 discord-canary.sh /app/bin/discord-canary",
+                "install -Dm755 discord-canary.sh /app/bin/com.discordapp.DiscordCanary",
                 "install -Dm755 disable-breaking-updates.py /app/bin",
                 "mv DiscordCanary /app/discord-canary",
                 "chmod +x /app/discord-canary/DiscordCanary",
                 "install -d /app/share/applications",
-                "sed -e 's/Icon=discord-canary/Icon=com.discordapp.DiscordCanary/' -e 's|Exec=/usr/share/discord-canary/DiscordCanary|Exec=discord-canary|' /app/discord-canary/discord-canary.desktop > /app/share/applications/${FLATPAK_ID}.desktop",
+                "sed -e 's/Icon=discord-canary/Icon=com.discordapp.DiscordCanary/' -e 's|Exec=/usr/share/discord-canary/DiscordCanary|Exec=com.discordapp.DiscordCanary|' /app/discord-canary/discord-canary.desktop > /app/share/applications/${FLATPAK_ID}.desktop",
                 "install -Dm644 /app/discord-canary/discord.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
-                "install -Dm644 com.discordapp.DiscordCanary.metainfo.xml /app/share/metainfo/com.discordapp.DiscordCanary.metainfo.xml"
+                "install -Dm644 com.discordapp.DiscordCanary.appdata.xml /app/share/appdata/com.discordapp.DiscordCanary.appdata.xml",
```

Comment:
This was correct, appdata is the old filename and location https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename